src/org/usfirst/frc/team694/robot/commands/CrabArmFlapAndSpatulaFlipDownCommand.java

```diff
@@ -0,0 +1,19 @@
+package org.usfirst.frc.team694.robot.commands;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+import edu.wpi.first.wpilibj.command.TimedCommand;
+import edu.wpi.first.wpilibj.command.WaitCommand;
+
+/**
+ *
+ */
+public class CrabArmFlapAndSpatulaFlipDownCommand extends CommandGroup {
+    private static final int timeDelayForSpatulaFlip = 2;
```

Comment:
This should be more specific in its name, and at least reference the units of time being used.